integration/raft/config_test.go

```diff
@@ -391,20 +384,41 @@ var _ = Describe("EndToEnd reconfiguration and onboarding", func() {
 				return config
 			})
 
-			By("Waiting for orderer3 to see the leader")
-			leader := findLeader([]*ginkgomon.Runner{ordererRunners[2]})
-			leaderIndex := leader - 1
-
-			fmt.Fprint(GinkgoWriter, "Killing the leader", leader)
-			ordererProcesses[leaderIndex].Signal(syscall.SIGTERM)
-			Eventually(ordererProcesses[leaderIndex].Wait(), network.EventuallyTimeout).Should(Receive())
+			By("Adding the third orderer to the channel")
+			addConsenter(network, peer, orderer, "systemchannel", etcdraft.Consenter{
+				ServerTlsCert: thirdOrdererCertificate,
+				ClientTlsCert: thirdOrdererCertificate,
+				Host:          "127.0.0.1",
+				Port:          uint32(network.OrdererPort(orderer3, nwo.ClusterPort)),
+			})
 
-			By("Ensuring orderer3 detects leader loss")
-			leaderLoss := fmt.Sprintf("Raft leader changed: %d -> 0", leader)
-			Eventually(ordererRunners[2].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say(leaderLoss))
+			By("removing orderer3's TLS root CA certificate")
```

Comment:
It's too early to remove that consenter. You just added it and you are immediately removing it from the config. 

We need to wait for the consenter to onboard, and make sure it participates in consensus and only then we need to remove it. 
Otherwise it might be a false positive test execution.

Reply:
I've updated this by waiting for orderer3 to see the leader but let me know if there's anything else we can/should do here.